internal/processor/processor.go

```diff
@@ -26,7 +26,7 @@ type Processor struct {
 	jobPartitionUUIDRegex *regexp.Regexp
 }
 
-func New(
```

Comment:
Why the name change ? We are in the processor package and there is only 1 `New` function, so `processor.New` is already explicit

Reply:
For clients, we have `NewClient`, in the Nozzle, we have `NewNozzle` so i wanted to do the same here.